system-tests/lib/cre/environment/environment.go

```diff
@@ -187,16 +187,11 @@ func SetupTestEnvironment(
 		return nil, pkgerrors.Wrap(topoErr, "failed to build topology")
 	}
 
-	gatewayJobConfigs, gErr := gateway.JobConfigs(
-		deployedBlockchains.RegistryChain().CtfOutput(),
-		topology,
-		updatedNodeSets,
-		input.GatewayWhitelistConfig,
-	)
+	gatewayConfigs, gErr := gateway.Configs(topology)
 	if gErr != nil {
-		return nil, pkgerrors.Wrap(gErr, "failed to build gateway job config")
+		return nil, pkgerrors.Wrap(gErr, "failed to build gateway configs")
 	}
-	topology.GatewayJobConfigs = gatewayJobConfigs
+	topology.GatewayConfigs = gatewayConfigs
```

Comment:
@Tofel Why do we need to set this here? IIUC this is setting up the config that will be used to generate the job spec -- can we somehow move all Gateway configuration code to one place? If not, why not?

Reply:
done